.gitlab-ci.yml

```diff
@@ -0,0 +1,68 @@
+variables:
+  CONTAINER_REGISTRY:     "docker.io/parity"
+  CONTAINER_REPO:         "substrate-telemetry-backend"
+  KUBE_NAMESPACE:         "substrate-telemetry"
+  IMAGE_FULL_NAME:        "${CONTAINER_REGISTRY}/${CONTAINER_REPO}:${CI_COMMIT_SHORT_SHA}"
+
+stages:
+  - dockerize
+  - staging
+
+#.delete_deployment:       &delete_deployment    
+#  - helm uninstall -n "$KUBE_NAMESPACE" "$KUBE_NAMESPACE"
+
+.deploy-k8s:              &deploy-k8s
+  image:                  paritytech/kubetools:3.5.3
+  script:
+    - echo "Deploying using image $IMAGE_FULL_NAME"
+    - echo "Using Helm `helm version --short`"
+    - export KUBERNETES_VERSION_TAG="$CI_PIPELINE_ID"
+    - |-
+      sed -i "s/version:.*/version: $KUBERNETES_VERSION_TAG/" ./helm/Chart.yaml
```

Comment:
i think the version of the chart shouldn't change with the kubernetes version. is `appVersion` the version of the deployed app, so it should be replaced with the version of `substrate-telemetry`?

Reply:
good catch.
remove that line.